documentation/reference/consent_decision_making.md

```diff
@@ -0,0 +1,570 @@
+  and are difficult to reverse and not strictly necessary to accomplish the
+  goals of the project
+- "Ratified proposal": A proposal that has undergone the formal decision-making
+  process and has been accepted as a valid solution for the problem it tries to
+  address by the participants of the discussion.
+- "Process proposal": A proposal to change processes followed by the
+  contributors in the course of stewarding the project towards success.
+- "Purpose": The motivation for the proposal. This is typically a statement
+  describing the problem the proposal aims to solve. In the original Sociocratic
+  model this is called the "driver".
+
+## Major differences from the original process
+
+Aside from being conducted primarily as an asynchronous process, Openverse's
+consent decision-making process has the following major differences from the
+Sociocratic model.
+
+### Reactions
+
+While the original process has a separate reactions round, in Openverse's
+process, reactions can be shared at any time. Be careful to ensure that they're
+not objections (which should, with few exceptions, be reserved for the objection
+round). Examples of reactions are:
+
+- "I love this approach"
+- "I like this proposal, but I don't understand X part of it yet"
+- "I don't think this will work, but I left some questions to clarify the areas
+  I have concerns over"
+- "I don't understand the underlying problem, so the solution seems too abstract
+  to me"
+- "I don't think this solution will solve the problem as described"
+
+Note that these are not questions (they do not ask for clarification) nor are
+they objections (they do not point to specific problems with the proposed
+solution).
+
+### Removal of initial rounds
+
+Because our process is primarily asynchronous, because our decision-making
+happens in public, and because the outcomes of our discussions are ultimately
+documented in GitHub or other accessible venues, we can skip some of the initial
+rounds. Specifically, the "consent to driver" and the "present the proposal"
+rounds cannot work in the same way. Typically, the purpose of a proposal will
+have been clarified through a previous discussion. Proposals raised without
+prior "buy-in" from other contributors should be done so carefully, with the
+understanding that participants may question the purpose and need for the
+proposal. The presentation of a proposal happens implicitly when the GitHub PR
+or discussion is opened to contain the proposal, so we do not need a special
+step for this.
+
+## Process summary
+
+Full descriptions of each of these rounds of discussion are below. This section
+is meant as a summary for getting an initial overview for people new to the
+process or refreshing the broad intentions for folks already familiar with it.
+
+These steps are ordered and must be followed in the order they are presented
+here.
+
+> **Nota bene**: The "goal" column is a very brief summary and is not intended
```

Comment:
"Note" instead maybe? Bonus points for the 

```md
> **Note**
> The "goal"column...
```

syntax which renders specially in GitHub flavoured markdown:

> **Note**
> The "goal"column...